openwisp_monitoring/notifications/tests/test_notifications.py

```diff
@@ -24,31 +24,36 @@ def test_general_check_threshold_crossed_immediate(self):
         admin = self._create_admin()
         m = self._create_general_metric(name='load')
```

Comment:
good chance to move this entirely to `openwisp_monitoring.monitoring.tests.test_notifications.py`